doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -6,54 +6,62 @@ How to generate an LSST reference catalog
 
-The process for generating an LSST-style HTM indexed reference catalog is similar to that of running other LSST Tasks: write an appropriate ``Config`` and run :lsst-task:`~lsst.meas.algorithms.ingestIndexReferenceTask.IngestIndexedReferenceTask`
-The differences are in how you prepare the input data, what goes into that ``Config``, how you go about running the ``Task``, and what you do with the final output.
+The process for generating an LSST-style HTM indexed reference catalog is similar to that of running other LSST Tasks: write an appropriate ``Config`` and run :lsst-task:`~lsst.meas.algorithms.convertReferenceCatalog.ConvertReferenceCatalogTask`
+The differences are in how you prepare the input data, how you go about running the ``Task``, and what you do with the final output.
 
 Ingesting a large reference catalog can be a slow process.
 Ingesting all of Gaia DR2 took a weekend on a high-performance workstation running with 8 parallel processes, for example.
 
 This page uses `Gaia DR2`_ as an example.
 
+.. note::
+
+    If you have an already existing converted reference catalog on disk (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers), you can skip the first few steps, and just :ref:`register and ingest the files <lsst.meas.algorithms-refcat-ingest>` directly, after creating a suitable filaname->htm7-index astropy table file.
+
 .. _Gaia DR2: https://www.cosmos.esa.int/web/gaia/dr2
 
 1. Gathering data
 =================
 
-:lsst-task:`~lsst.meas.algorithms.ingestIndexReferenceTask.IngestIndexedReferenceTask` reads reference catalog data from one or more text or FITS files representing an external catalog (e.g. :file:`GaiaSource*.csv.gz`).
-In order to ingest these files, you must have a copy of them on a local disk.
+:lsst-task:`~lsst.meas.algorithms.convertReferenceCatalog.ConvertReferenceCatalogTask` reads reference catalog data from one or more text or FITS files representing an external catalog (e.g. :file:`GaiaSource*.csv.gz`).
+In order to convert these files, you must have a copy of them on a local disk.
 Network storage (such as NFS and GPFS) are not recommended for this work, due to performance issues involving tens of thousands of small files.
 Ensure that you have sufficient storage capacity.
-For example, the GaiaSource DR2 files take 550 GB of space, and the ingested LSST reference catalog takes another 200 GB.
+For example, the GaiaSource DR2 files take 550 GB of space, and the converted LSST reference catalog takes another 200 GB.
 
-To write the config, you will need to have a document that describes the columns in the input data, including their units and any caveats that may apply (for example, Gaia DR2 does not supply magnitude errors).
+To write the config for the converter, you will need to have a document that describes the columns in the input data, including their units and any caveats that may apply (for example, Gaia DR2 does not supply magnitude errors).
```

Comment:
The fluxes and flux errors they supply are e-/s, which are not something we can easily use, hence the special-cased `ConvertGaiaManager`.

You are correct that the Gaia-supplied magnitudes are Vega, and they aren't actually used. `mag_column_list` is a required config field and is used to populate the output schema, which is why it exists here in the docs. I'll update the comment attached to it.

Reply:
I think I had too many comments on this which led to some confusion.  I understand why `ConvertGaiaManager` is necessary.  I don't think the comment that "Gaia DR2 does not supply magnitude errors" is relevant, since we don't use the magnitudes either.  Maybe "(for example, with Gaia DR2 we convert native fluxes and errors in e-/s to AB magnitudes".